client/src/helpers/utils/checkIfUndefined.ts

```diff
@@ -0,0 +1,10 @@
+export function checkIfUndefined<T>(
```

Comment:
Looks good! 🙌🏼  Though, we can rename this so that it conveys what it does a bit more clearly, e.g:
```suggestion
export function assertNotNullOrUndefined<T>(
```
Especially because it can `throw`, and doesn't return a `boolean` 👍🏼

Reply:
Sure, I will do that, Thanks